src/bindgen/utilities.rs

```diff
@@ -39,6 +39,28 @@ pub fn find_first_some<T>(slice: &[Option<T>]) -> Option<&T> {
     None
 }
 
+pub trait SynItemFnHelpers: SynItemHelpers {
+    fn export_name_attr(&self) -> Option<String>;
+
+    fn exported_name(&self) -> Option<String>;
+}
+
+impl SynItemFnHelpers for syn::ItemFn {
+    fn export_name_attr(&self) -> Option<String> {
```

Comment:
Given this only has one caller, probably worth inlining it instead of adding the function? It's a one liner anyway.

Reply:
Done